src/framework/audio/internal/platform/lin/linuxaudiodriver.cpp

```diff
@@ -92,8 +92,11 @@ static void alsaCleanup()
     if (s_alsaData->threadHandle) {
         pthread_join(s_alsaData->threadHandle, nullptr);
     }
-    snd_pcm_drain(s_alsaData->alsaDeviceHandle);
-    snd_pcm_close(s_alsaData->alsaDeviceHandle);
+    qDebug("alsa handle: %p", s_alsaData->alsaDeviceHandle);
```

Comment:
I think using qDebug here is not preferred. Better use `LOGD() << "alsa handle: " << s_alsaData->alsaDeviceHandle`, or remove the logging if it is not necessary.

Reply:
Thanks! Forgot to remove debugging.